Controllers/UserAuthenticationController.php

```diff
@@ -0,0 +1,65 @@
+<?php
+namespace Controllers;
+use models\User;
+
+class UserAuthenticationController extends BasicController{
+
+	public function __construct() {
+		parent::__construct();
```

Comment:
Entire file uses too many white spaces. In php we should have 2 spaces for indentation.